Dockerfile.osx

```diff
@@ -38,8 +38,10 @@ RUN git clone https://github.com/mono/mono --branch ${mono_version} --single-bra
     ln -s /usr/bin/mono /root/dependencies/mono/bin/mono && \
     ln -s /usr/bin/mono-sgen /root/dependencies/mono/bin/mono-sgen && \
     ln -sf /usr/lib/mono/* /root/dependencies/mono/lib/mono && \
-    cp -rvp /etc/mono /root/dependencies/mono/etc && \
-    cp /root/files/mono-config-macosx /root/dependencies/mono/etc/config && \
```

Comment:
We copied this file to the wrong location, it's meant to be in `${prefix}/etc/mono/config`.
The in-container build script then used the wrong path too so it worked, but it's better to use the expected path in the mono distribution to avoid confusion (and I back up the old file for comparison, as new Mono versions sometimes add content there).